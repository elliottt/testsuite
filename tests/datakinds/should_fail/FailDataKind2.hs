module FailDataKind2 where

-- Fails because of no -XDataKinds
data kind Foo = Bar
