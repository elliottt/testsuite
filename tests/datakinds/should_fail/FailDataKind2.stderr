
FailDataKind2.hs:4:6:
    Malformed head of type or class declaration: kind Foo
